hosting/static/hosting/css/commons.css

```diff
@@ -1,22 +1,22 @@
 .dashboard-container {
-	padding-top:70px; padding-bottom: 70px;
-	width: 90%;
-	margin: 0 auto;
-	max-width: 768px;
+  padding-top:70px; padding-bottom: 70px;
```

Comment:
Not a big deal but it would be great if we could reformat the code in this page.